lib/src/androidTest/java/space/taran/arklib/TestingUtils.kt

```diff
@@ -0,0 +1,9 @@
+package space.taran.arklib
+
+import dev.arkbuilders.arklib.ResourceId
+
+fun randomResourceId():ResourceId =
+    ResourceId(
+        (Math.random() * 1000).toLong() + 1,
```

Comment:
Better to use `Random.nextLong` method and also would be good to cover all range of Long, not just first 1000 values.

Reply:
@kirillt the issue with Random.nextLong is that it gives a value between Long.MIN_VALUE and Long.MAX_VALUE meaning it can return zero. Can the value of score be negative?

Reply:
@tanaytandon12 Good point. Scores can be negative, yes, but for scores we don't really care. CRC-32 and filesize can have only positive values.

Reply:
https://stackoverflow.com/questions/2546078/java-random-long-number-in-0-x-n-range
> Starting from Java 7 (or Android API Level 21 = 5.0+) you could directly use ThreadLocalRandom.current().nextLong(n) (for 0 ≤ x < n) and ThreadLocalRandom.current().nextLong(m, n) (for m ≤ x < n).

Reply:
In Kotlin, it's just `Random.nextLong(1, Long.MAX_VALUE)`